lib/worker.js

```diff
@@ -87,5 +89,13 @@
 		}, 1000).unref();
 	};
 
+	Worker.prototype.sendCounterToFluentd = function(name, value) {
+		if (Object.keys(this.kardiaConfig).length === 0) {
+			this.kardiaConfig = require(__dirname + '/../index.js').config;
```

Comment:
You cannot do this because the config object is not set in the index.js. This is because you are requiring it from a worker process — and the config is only set in the master process.